src/vuex/modules/minting.js

```diff
@@ -0,0 +1,41 @@
+export default ({ node }) => {
+  const emptyState = {
+    pool: {},
+    loading: false,
+    loaded: false,
+    error: null
+  }
+  const state = {
+    ...emptyState
+  }
+  const mutations = {
+    setAnnualProvision(state, annualProvision) {
+      state.annualProvision = annualProvision
+    }
+  }
+
+  const actions = {
+    async getMintingParameters({ state, commit, rootState }) {
```

Comment:
any reason you didn't call this `getProvisionParameters`?

Reply:
The module is called `minting`